src/views/Teams/Team.js

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { getTeamById } from '../../services/teams';
+import './Teams.css';
+
+export default function Team(props) {
+  const [team, setTeam] = useState([]);
```

Comment:
Originally we had useState({ players: [] })
This meant that our initial team state would instantiate as an empty player array. 
This would allow us to bypass the loading function. If our initial state contains an empty player array, then line 23 won't break because it, at least, has an empty array.